doc-content/enterprise-only/springboot/spring-boot-persistence-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='spring-boot-persistence-proc_{context}']
+= Configuring plugable variable persistence
+
+You can provide an arbitrary entity manager for configured process variable persistence in your {PRODUCT} Spring Boot application. To do this, add named beans during the object marshalling strategy resolution. This configuration defines a second entity manager factory based on a qualifier second data source. This process will not interfere with the primary data source.
+
+.Prerequisites
+
+* You have an existing {PRODUCT} Spring Boot project.
+
+.Procedure
+
+. Add the following `auditEntityManager` JavaBean to your java class:
++
+[source]
+----
+    @Bean(name = "auditEntityManager")
+    @ConditionalOnMissingBean(name = "auditEntityManager")
+    public LocalContainerEntityManagerFactoryBean entityManagerFactory(@Qualifier("jpaAuditDataSource") DataSource dataSource, JpaProperties jpaProperties) {
+        return EntityManagerFactoryHelper.create(applicationContext,
+                                                 dataSource,
+                                                 jpaProperties,
+                                                 AUDIT_PERSISTENCE_UNIT_NAME,
+                                                 PERSISTENCE_XML_LOCATION);
```

Comment:
same as above
```suggestion
                                                 "classpath:/META-INF/persistence.xml");
```

Reply:
OK so is your suggestion here a concrete example? Or should we make it a variable?

Reply:
it's a concrete example, so the reader can easily identify what we are actually passing here rather than using a variable/constant as it was previously - hope it makes sense?